llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -7197,6 +7199,48 @@ SDValue SelectionDAG::getNode(unsigned Opcode, const SDLoc &DL, EVT VT,
       return N1.getOperand(1);
     break;
   }
+  case ISD::MCOMPRESS: {
+    EVT VecVT = N1.getValueType();
+    EVT MaskVT = N2.getValueType();
+    (void)MaskVT;
```

Comment:
Nit you can use maybe_unused instead to prevent warnings
```suggestion
    [[maybe_unused]] EVT MaskVT = N2.getValueType();
```

Reply:
okay, changed. I thought of this but did not find any use of `[[maybe_unused]]` in this file, so I stuck with the file's convention.

Reply:
Oh sorry didn't realise that's what the rest of the file was using. We should probably mop up the other void uses now that we've migrated to C++17 (in another PR!)